proving/postrs/bridge.go

```diff
@@ -0,0 +1,54 @@
+package postrs
+
+// #cgo LDFLAGS: -L/home/bartosz/workspace/post-rs/target/release-clib -lpost
+// #include <stdlib.h>
+// #include "prover.h"
+import "C"
+
+import (
+	"fmt"
+	"unsafe"
+
+	"github.com/spacemeshos/post/config"
+)
+
+type Proof struct {
+	Nonce    uint32
+	Indicies []uint64
+}
```

Comment:
It's a different type. It contains indices as uint64s that are later compressed. That compression is not implemented on Rust side (I'm not even sure if it's worth implementing there, at least now).

Reply:
Since this type is only required internally we could probably make it private (the Public fields are still accessible from other packages). Just to avoid confusion between `shared.Proof` and `postrs.Proof`.